tests/unittests/Foundation/NSFileManagerTests.mm

```diff
@@ -27,7 +28,8 @@
 #include "TestUtils.h"
 
 TEST(NSFileManager, GetAttributes) {
-    NSString* testFileFullPath = getPathToFile(@"/data/NSFileManagerUT.txt");
+    NSString* testFileFullPath =
+        [[NSString stringWithUTF8String:GET_CURRENT_TEST_DIRECTORY().c_str()] stringByAppendingPathComponent:@"/data/NSFileManagerUT.txt"];
```

Comment:
I'm not sure what you mean... this is a std::string returning thing. The thing is just a bit weird. Are you wanting a more formalized class or something? I agree that the macro "function" isn't super clean ...